modules/appeals_api/spec/support/rswag_config.rb

```diff
@@ -235,10 +235,9 @@ def schemas(api_name: nil)
       merge_schemas(
         sc_create_schemas,
         sc_response_schemas,
-        sc_alt_signer_schemas,
-        contestable_issues_schema.slice(*%i[contestableIssue]),
-        generic_schemas.except(*%i[errorWithTitleAndDetail timeStamp uuid X-Consumer-ID X-Consumer-Username X-VA-NonVeteranClaimant-SSN X-VA-NonVeteranClaimant-Birth-Date]),
-        shared_schemas.slice(*%w[address phone timezone nonBlankString])
+        appealable_issues_schema.slice(*%i[appealableIssue]),
+        generic_schemas.slice(*%i[errorModel documentUploadMetadata]),
+        shared_schemas.slice(*%w[address icn phone timezone nonBlankString])
```

Comment:
I noticed here that we were still using contestable issues shared schemas in these docs 😬 I'll be fixing anything else I see like this as I go through the APIs